packages/gitbook-docs/src/__tests__/parseContent.test.ts

```diff
@@ -29,6 +29,14 @@ const config = {
       path: '/to/s1',
       title: 'S1 Home',
     },
+    {
+      path: '/to/s0/ente-creditore/modalita-dintegrazione/integrazione-touch-point-dellec-con-checkout',
+      title: 'Integrazione touch point dell’EC con Checkout',
+    },
+    {
+      path: '/to/s0/comunicare-un-servizio/modificare-o-ampliare-un-servizio',
+      title: 'Modificare o ampliare un servizio',
```

Comment:
These two examples are redundant. To keep the tests simple, I would have used the [existing paths](https://github.com/pagopa/developer-portal/pull/590/files#diff-1f878dc19510fc231071b13b3bb428deeecc7b828ba093f1197592a54f3f3f8bR20-R31).

Reply:
Done